MovieChecker/MovieChecker/MovieDetailViewController.h

```diff
@@ -0,0 +1,17 @@
+//
+//  MovieDetailViewController.h
+//  MovieChecker
+//
+//  Created by Joseph Goldberg on 1/22/15.
+//  Copyright (c) 2015 Vokal. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+#import "Movie.h"
+
+@interface MovieDetailViewController : UITableViewController
+
+@property (nonatomic, weak) Movie *movie;
```

Comment:
I would make this a `strong` reference—it wouldn't make sense for the view controller to continue on without the movie model object.